llvm/lib/CodeGen/GlobalISel/IRTranslator.cpp

```diff
@@ -2688,7 +2688,7 @@ bool IRTranslator::translateKnownIntrinsic(const CallInst &CI, Intrinsic::ID ID,
 
 bool IRTranslator::translateInlineAsm(const CallBase &CB,
                                       MachineIRBuilder &MIRBuilder) {
-  if (containsBF16Type(CB))
+  if (!MF->getTarget().getTargetTriple().isSPIRV() && containsBF16Type(CB))
```

Comment:
NIT: Can we add a helper `canHandleBF16ForTarget()` that does the `MF->getTarget().getTargetTriple().isSPIRV()` check?

Adding a hook in `TargetMachine` would be ideal but it'd be an overkill.